content/source-git/status.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Packit’s statement on Source-git for 2023/Q3
+date: 2023-06-22T23:59:59+01:00
+weight: 2
+---
+# Packit’s statement on Source-git for 2023/Q3
+
+The Packit team has worked on the source-git project for a few years (with varying priority) and a lot of research, prototyping and coding has been done. The goal of this document is to provide a status update on the source-git project and set clear expectations for the future.
+
+It’s a workflow to efficiently maintain software downstream using the upstream git history with layering custom content on top as git commits and respective RPM spec file patches.
```

Comment:
```suggestion
Source git is a workflow to efficiently maintain software downstream using the upstream git history with layering custom content on top as git commits and respective RPM spec file patches.
```

Reply:
Let me update the source-git spelling to be consistent with the Maťo's changes..

```suggestion
Source-git is a workflow to efficiently maintain software downstream using the upstream git history with layering custom content on top as git commits and respective RPM spec file patches.
```